frontend/E2ETests/test/steps/projectDetails.steps.js

```diff
@@ -1,9 +1,11 @@
 import { When, Then } from 'cucumber';
+import Tabs from '../pages/Components/Tabs/tabs.component';
```

Comment:
I don't see the usage of this in this file.

Reply:
good catch, I ended up removing code imported from that file 👍